docs/source/how-to-upstream.mdx

```diff
@@ -89,6 +119,7 @@ The [`Repository`] class also has a [`~Repository.push_to_hub`] function to add
 
 ```python
 >>> from huggingface_hub import Repository
+>>> repo = Repository(local_dir="path/to/local/repo")
```

Comment:
Here, I don't understand why you specify a local directory and then do a git pull.

Why would you need to git pull in case there's no remote specified?

Reply:
And again here, would be useful to explain whether a remote repository needs to exist already, or whether it will be created automatically in case it doesn't.

Reply:
Hmm, my understanding of `Repository` is that if I have an existing local clone, I can just initialize it from the `local_dir`. Then I can `git_pull` to update it with the repository on the Hub.

Let me know if I am misunderstanding how that works! :)

Reply:
Ok in that case I would add a comment to clarify this, maybe replace `path/to/local/repo` with `path/to/local/clone` in the code snippet above

Reply:
Oh nvm, just saw you added additional text above the code snippet. Fine for me!